### ISSUE TITLE
Consider renaming proteus

### ISSUE THREAD

**Issue by hellais** [ooni/orchestra#17]
As noted by @darkk (and I actually also experienced this), proteus has a very small Levenshtein distance from prometheus and leads it to be quite confusing.

Maybe we should consider renaming it to something a bit more unique. I think in general it would be cool to use for all ooni software components a name that follows the marine theme.

As such I would propose to rename it to [Oarfish](https://en.wikipedia.org/wiki/Oarfish), "known in Japanese folklore as the Messenger from the Sea God's Palace". It also is somewhat closer to "Orchestration" as a name.

**Comment by darkk** [comment]
How painful is the rename? I mean, where is the string hardcoded?

The article you linked links two wonderful pages: [List of common fish names](https://en.wikipedia.org/wiki/List_of_common_fish_names) and [List of fish families](https://en.wikipedia.org/wiki/List_of_fish_families). These two bring us [notograptus](https://en.wikipedia.org/wiki/Notograptus) sending notifications :-)

**Comment by bassosimone** [comment]
> How painful is the rename? I mean, where is the string hardcoded?

It probably is in many places, but I think we can solve that with recursive grep and sed.

I do like both names. I _think_ oarfish may be better because it's easier to spell (we renamed libight to measurement-kit because it was difficult to spell and communicate).

FTR, I also did notice the names similarity and have been confused a couple of times. I think renaming the project is a good thing to do. If we want to be more serious, the name could be something like `ooni-orchestration`, which conveys the actual purpose of the repository. (I don't like this option that much, but I am mentioning it anyway, since it's a third option.)
